zds/settings.py

```diff
@@ -509,7 +509,11 @@
         'maximum_slug_size': 150,
         'sec_per_minute': 1500,
         'editorial_line_link':
-        u'https://zestedesavoir.com/articles/222/la-ligne-editoriale-officielle-de-zeste-de-savoir/'
+        u'https://zestedesavoir.com/articles/222/la-ligne-editoriale-officielle-de-zeste-de-savoir/',
+        'sec_per_minute': 1500,
+        'list': 'new',  # put 'old' if you want to go back to old way of displaying tutorial
+        'selected_content_method_name': 'get_most_commented_list',  # allow to customize "top tutorial" appears
+        'content_per_theme': 10
```

Comment:
*Note to self*: ces paramètres ne sont utilisés nul part. Il faudrait ^^ (mais qu'est ce que `'selected_content_method_name'` et `'list'` sont sencé faire ?!?)